### PR TITLE
{lib}[GCCcore/11.2.0] Blosc2 v2.4.3

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Blosc2/Blosc2-2.4.3-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True, 'cstd': 'c++11'}
+
+source_urls = ['https://github.com/Blosc/c-blosc2/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d4aa5e0794598794f20ab950e973d44f0d0d9c133ea1a5a07cb200fa54d2e036']
+
+builddependencies = [
+    ('binutils', '2.38'),
```
Comment (bedroge) [inline easybuild/easyconfigs/b/Blosc2/Blosc2-2.4.3-GCCcore-11.2.0.eb:21]:
```suggestion
    ('binutils', '2.37'),
```

Comment (sassy-crick) [reply easybuild/easyconfigs/b/Blosc2/Blosc2-2.4.3-GCCcore-11.2.0.eb]: Thanks. I just corrected that.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/df20d93d366b609300fd6256522ab19b for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**